### PR TITLE
Docker local storage volume demo not working properly

### PR DESCRIPTION
**Pull request by XethalKhan**
Unable to login with jpicado account. 

1. Fixed reading proper htpasswd file in config.yaml.
2. Added **version: '3'** in docker-compose.yaml file.

### CODE REVIEW

File: docker-examples/v6/docker-local-storage-volume/conf/config.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ storage: /verdaccio/storage
 
 auth:
   htpasswd:
-    file: /verdaccio/conf/htpasswd
+    file: htpasswd
```
Comment (juanpicado) [inline docker-examples/v6/docker-local-storage-volume/conf/config.yaml:18]: Thanks, I'm just curious how did you figure out this Is the right setup? what was the error you were getting?

Comment (XethalKhan) [reply docker-examples/v6/docker-local-storage-volume/conf/config.yaml:18]: After launching docker container, I wasn't able to login with default jpicado default account. GUI showed message "Unable to sign in", and I also saw in network tab HTTP 401 response with error message "bad username/password, access denied".

I checked out [configuration docs](https://verdaccio.org/docs/configuration) and compared default config.yaml file with dockers example config.yaml file. Default configuration in docs suggests that htpasswd file location should be relative to verdaccio/config directory, which was not the case in dockers config.yaml file.

Comment (juanpicado) [reply docker-examples/v6/docker-local-storage-volume/conf/config.yaml:18]: Ok, the `default.yaml` and `docker.yaml` are not exactly the same, the unique difference are
- auth -> htpasswd -> file
- storage
- plugins

https://github.com/verdaccio/verdaccio/tree/master/packages/config/src/conf

But does is only an special configuration for the Dockerfile, https://github.com/verdaccio/verdaccio/blob/master/Dockerfile#L54 that's it, so you should relies on what the dockerfile indicates.

On this example the `config.yaml` indicates `/verdaccio/conf/htpasswd` and the docker compose is mapping volume

```
    volumes:
      - './storage:/verdaccio/storage'
      - './conf:/verdaccio/conf'
```

So I think the configuration is correct, you have permissions issues,  check docs https://verdaccio.org/docs/docker#running-verdaccio-using-docker

I think applying more permissions to the folder locally can solve your problem

Comment (XethalKhan) [reply docker-examples/v6/docker-local-storage-volume/conf/config.yaml:18]: Have you tried running example based on instructions in [README file](https://github.com/verdaccio/verdaccio/blob/master/docker-examples/v6/docker-local-storage-volume/README.md)

Generally, when people go to example directory, they expect examples to work after following instructions. If it is requiered to change permissions or change config.yaml file, it should be specified in README above.

This commit is a result of debugging, and problem was not with permissions.

**Comment by juanpicado** [comment]
I think the examples were affected for a bug I've found at https://github.com/verdaccio/verdaccio/commit/09753cc1fa7777c9deab514ab95e8931728382ea so I guess we can close this, if still happens please open a ticket.

**Pull request closed without merging**